src/provable/gen/js-layout.ts

```diff
@@ -338,7 +338,9 @@ let jsLayout = {
                         },
                         timestamp: {
                           type: 'option',
-                          optionType: 'flaggedOption',
+                          optionType: 'closedInterval',
```

Comment:
changes here are caused by the Mina companion: https://github.com/MinaProtocol/mina/pull/12162